src/components/DeploymentStatusIcon.vue

```diff
@@ -23,6 +23,8 @@
         return 'One or more processes are actively polling this deployment'
       case 'not_ready':
         return 'This deployment has not been polled in the past 60 seconds'
+      case 'disabled':
+        return 'This deployment is disabled and will not be polled'
```

Comment:
This probably applies to the `not_ready` status above as well but I find this wording to be particularly unclear; will work still build up (e.g. late runs)? I don't have a great suggestion (sorry) but would like if we could make this a little clearer, I don't quite know what the behavior is though

Reply:
This is a great note -- I can definitely clean this language up

Reply:
That's already a lot cleaner! How does this behavior impact existing runs?

Reply:
Doesn't mess with them at all, only prevents new runs